pkg/testing/multipass/provisioner.go

```diff
@@ -170,6 +182,56 @@ func (p *provisioner) launch(ctx context.Context, cfg runner.Config, batch runne
 	return nil
 }
 
+func (p *provisioner) ensureInstanceNotExist(ctx context.Context, batch runner.OSBatch) error {
+	var output bytes.Buffer
+	proc, err := process.Start("multipass",
+		process.WithContext(ctx),
+		process.WithArgs([]string{"list", "--format", "json"}),
+		process.WithCmdOptions(
+			runner.AttachOut(&output),
+			runner.AttachErr(&output)))
```

Comment:
If we are going to parse the output, we probably don't want stderr in the mix, so we should use separate buffers